metadata_service/config.py

```diff
@@ -64,6 +64,9 @@ class Config:
     # On User detail method, these keys will be added into amundsen_common.models.user.User.other_key_values
     USER_OTHER_KEYS = {'mode_user_id'}  # type: Set[str]
 
+    # Number of minimum reader count to qualify for popular table
+    POPULAR_TABLE_MINIMUM_READER_COUNT = 10  # type: int
```

Comment:
```python
POPULAR_TABLE_MINIMUM_READER_COUNT = os.environ.get('POPULAR_TABLE_MINIMUM_READER_COUNT', 10)  # type: int
```
It should have been this. 🙃

Reply:
not sure, but we could override it through different config right?

Reply:
Right, production deployment should have some other config class. False alarm